pymedphys/_dicom/anonymise/__init__.py

```diff
@@ -64,6 +64,7 @@ def anonymise_dataset(  # pylint: disable = inconsistent-return-statements
     delete_unknown_tags=None,
     copy_dataset=True,
     replacement_strategy=None,
+    alternative_identifying_keywords=None,
```

Comment:
Can we just call this `identifying_keywords` throughout? And then, create an `lru_cached` function called `get_default_identifying_keywords` which then runs the following and returns the default keywords:

https://github.com/sjswerdloff/pymedphys/blob/cd15371571d29d88ad1db307c2ef8eb449311e77/pymedphys/_dicom/anonymise/__init__.py#L41-L44

We can then have `anonymised_dataset` be such that should it be `None`, then it sets `idetifying_keywords` to the default function.

What do you think?

Reply:
Will do.